src/WebJobs.Script/Description/Workers/WorkerFunctionDescriptorProvider.cs

```diff
@@ -100,5 +114,26 @@ protected override IFunctionInvoker CreateFunctionInvoker(string scriptFilePath,
 
             return parameters;
         }
+
+        internal bool BindingAttributeContainsExpression(IEnumerable<FunctionBinding> bindings)
+        {
+            foreach (ExtensionBinding binding in bindings)
```

Comment:
Any reason why you're using the derived ExtensionBinding type here rather than FunctionBinding? I think you should just use FunctionBinding, since there can be other subclasses, e.g. HttpBinding

Reply:
no reason, probably a typo! Fixed

Reply:
Ah I lied, there was a reason that I forgot about! 'FunctionBinding' does not contain a definition for 'Attributes' so I had to use ExtensionBinding here